pkg/helpers/helpers.go

```diff
@@ -0,0 +1,72 @@
+package helpers
+
+import (
+	"strings"
+
+	fluentbitconfig "github.com/calyptia/go-fluentbit-config"
+	"golang.org/x/exp/slices"
+)
+
+// pluginProps -
+// TODO: exclude already defined property.
+func PluginProps(kind, name string) []string {
```

Comment:
This should be moved to https://github.com/calyptia/go-fluentbit-config
For now move it to a package called like `fluentbit` since helpers doesn't denote anything, its the same as utils.